src/conventions/gmsh.jl

```diff
@@ -0,0 +1,102 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+Ordering convention of the GMSH project:
+https://gmsh.info/doc/texinfo/gmsh.html#Node-ordering
+"""
+abstract type GMSH <: OrderingConvention end
```

Comment:
I defined `GMSH` as an abstract type and dispatched on `::Type{GMSH}`, would there be any benefit of defining it as a normal struct? And, in that case, dispatch on an instance `GMSH()` rather than a `Type`?

Reply:
I think it is fine to dispatch on the abstract type. I am not aware of any issue.

Reply:
On the other hand, I think it is good to signal that this type is final. Can you make it a struct instead?

Reply:
I agree. I will do that tomorrow.